qfieldcloud_sdk/sdk.py

```diff
@@ -370,8 +559,15 @@ def list_jobs(
         job_type: JobTypes = None,
         pagination: Pagination = Pagination(),
     ) -> List[Dict[str, Any]]:
-        """
-        Returns a paginated lists of jobs accessible to the user.
+        """Return a paginated list of jobs accessible to the user.
```

Comment:
Everything should be paginated, no need to add this word here:
```suggestion
        """List project jobs.
```